_submissions/round-12/1/2015-02-26-for-loops-in-R.md

```diff
@@ -0,0 +1,15 @@
+---
+date: 2015-02-26
+round: Round 12
+title: Functions in R
+author: Laurie Baker
+permalink: /2015/02/LaurieBa/
+tags:
+  - Concept Map
+---
+
+Basic Functions in R
+
+Functions are useful tools that can be used to make coding tidier and more efficient. This map has been adapted from http://swcarpentry.github.io/r-novice-inflammation/02-func-R.html
+
+http://i.imgur.com/JcQnSVz.jpg
```

Comment:
This needs to be something like `![title](link)` for an image, but I'll fix that after I merge.